### PR TITLE
Bump @playwright/test from 1.30.0 to 1.34.1

### PR DESCRIPTION
**Pull request by baev**
#### Checklist
- [x] [Sign Allure CLA][cla]
- [ ] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

File: packages/allure-playwright/src/index.ts [hunk 1]
```diff
@@ -319,6 +319,10 @@ class AllureReporter implements Reporter {
     }
   }
 
+  printsToStdio(): boolean {
```
Comment (vovsemenv) [inline packages/allure-playwright/src/index.ts:322]: why we need this method?

File: packages/allure-playwright/test/fixtures.ts [hunk 2]
```diff
@@ -87,13 +87,18 @@ const runPlaywrightTest = async (
   if (additionalArgs) {
     args.push(...additionalArgs);
   }
-  const testProcess = fork(require.resolve("@playwright/test/cli"), args, {
+
+  const modulePath = require.resolve("@playwright/test/lib/cli");
+  await base.step(`${modulePath} ${args.join(" ")}`, () => {});
```
Comment (vovsemenv) [inline packages/allure-playwright/test/fixtures.ts:92]: why we doing this?

**Review by vovsemenv — APPROVED** [review]
Looks fine, small comments

File: packages/allure-playwright/src/index.ts (hunk 1 above)
Comment (baev) [reply packages/allure-playwright/src/index.ts:322]: From documentation:

```
 /**
   * Whether this reporter uses stdio for reporting. When it does not, Playwright Test could add some output to enhance
   * user experience. If your reporter does not print to the terminal, it is strongly recommended to return `false`.
   */
  printsToStdio?(): boolean;
```

File: packages/allure-playwright/test/fixtures.ts (hunk 2 above)
Comment (baev) [reply packages/allure-playwright/test/fixtures.ts:92]: To display executed commands in the report:

Screenshot 2023-05-26 at 10 43 36

**Pull request merged**